tests/src/main/java/com/hazelcast/simulator/tests/map/MapTimeToLiveTest.java

```diff
@@ -22,124 +22,145 @@
 import com.hazelcast.logging.Logger;
 import com.hazelcast.simulator.test.TestContext;
 import com.hazelcast.simulator.test.TestRunner;
-import com.hazelcast.simulator.test.annotations.Run;
+import com.hazelcast.simulator.test.annotations.RunWithWorker;
 import com.hazelcast.simulator.test.annotations.Setup;
 import com.hazelcast.simulator.test.annotations.Verify;
 import com.hazelcast.simulator.tests.map.helpers.MapOperationCounter;
 import com.hazelcast.simulator.utils.AssertTask;
-import com.hazelcast.simulator.utils.ThreadSpawner;
+import com.hazelcast.simulator.worker.selector.OperationSelectorBuilder;
+import com.hazelcast.simulator.worker.tasks.AbstractWorker;
 import com.hazelcast.spi.exception.DistributedObjectDestroyedException;
+import com.hazelcast.util.EmptyStatement;
 
-import java.util.Random;
 import java.util.concurrent.TimeUnit;
 
 import static com.hazelcast.simulator.utils.TestUtils.assertTrueEventually;
 import static org.junit.Assert.assertEquals;
 
 /**
- * In this test we are using map put methods with an Expire time.
- * we put keys at random into the map using sync and async methods with some proablity distribution
- * in the end we verify that the map is empty and all key value pairs have expired out of the map
+ * In this test we are using map put methods with an expire time.
+ *
+ * We put keys at random into the map using sync and async methods with some probability distribution.
+ * In the end we verify that the map is empty and all key value pairs have expired out of the map.
  */
 public class MapTimeToLiveTest {
 
     private static final ILogger LOGGER = Logger.getLogger(MapTimeToLiveTest.class);
 
+    private enum Operation {
+        PUT_TTL,
+        ASYNC_PUT_TTL,
+        GET,
+        ASYNC_GET,
+        DESTROY
+    }
+
     // properties
     public String basename = this.getClass().getSimpleName();
-    public int threadCount = 3;
     public int keyCount = 10;
 
-    // check these add up to 1
     public double putTTLProb = 0.4;
     public double putAsyncTTLProb = 0.3;
     public double getProb = 0.2;
     public double getAsyncProb = 0.1;
     public double destroyProb = 0.0;
 
-    public int maxTTLExpireyMs = 3000;
-    public int minTTLExpireyMs = 1;
+    public int maxTTLExpiryMs = 3000;
+    public int minTTLExpiryMs = 1;
```

Comment:
again - change of the properties name. tricky. 
@Danny-Hazelcast is it alright from your PoV?